constants/styles.js

```diff
@@ -30,3 +31,10 @@ export const presetColors = [
   colors.deep_green,
   colors.smokey_topaz,
 ]
+
+export const DEFAULT_AVATAR_STYLE = { width: 60, height: 60 }
+
+export const AVATAR_STYLE_WITH_COLOR = {
+  ...DEFAULT_AVATAR_STYLE,
+  backgroundColor: colors.columbia_blue,
+}
```

Comment:
I don't think we need these in a global area. They can just live in the `UserAvatar` component since they are unlikely to be shared outside of the avatar.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/196/commits/f307adf2b8668c103503147fc268b63c299b99c6